docs-ref-services/servicebus.md

```diff
@@ -15,34 +15,189 @@ ms.service: service-bus
 
 ## Overview
 
-Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging. 
+Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging.
+
+As of version 0.50.0 a new AMQP-based API is available for sending and receiving messages. This update involves breaking changes.
```

Comment:
Maybe a "What's new" header? And ideally, we'd point out why AMQP is better (or, in other words, tell people why the breaking changes were needed)